code/messaging/src/phone/java/com/adobe/marketing/mobile/Message.java

```diff
@@ -36,14 +36,14 @@
  */
 public class Message extends MessagingDelegate {
     private final static String SELF_TAG = "Message";
+    private final FullscreenMessage aepMessage;
+    private final Map<String, WebViewJavascriptInterface> scriptHandlers = new HashMap<>();
+    private final Handler webViewHandler = new Handler(MobileCore.getApplication().getMainLooper());
```

Comment:
why did we move these?

Reply:
android studio's code format moved these around... i can move them back

Reply:
possibly because of the final property?

Reply:
ah, cause it wants you to list private properties first i suppose.  was just curious :)